### PR TITLE
Search SDK: Updating SearchServiceClient spec per recent bug fixes

### PR DESCRIPTION
**Pull request by bitsforthought**
See individual commits for details.

We need to merge this to master in preparation for creating a new search-v2-rc branch.

### CODE REVIEW

File: search/2015-02-28-Preview/swagger/searchservice.json [hunk 1]
```diff
@@ -1241,7 +1241,7 @@
           "type": "integer",
           "format": "int32",
           "default": 255,
-          "description": "The maximum token length. Tokens longer than the maximum length are split. Default is 255."
+          "description": "The maximum token length. Default is 255. Tokens longer than the maximum length are split. The maximum token length that can be used is 300 characters."
```
Comment (tbombach) [inline search/2015-02-28-Preview/swagger/searchservice.json]: Could you add a `"maximum"` constraint to this property?

File: search/2015-02-28-Preview/swagger/searchservice.json [hunk 2]
```diff
@@ -1307,7 +1307,7 @@
...
```
Comment (tbombach) [inline search/2015-02-28-Preview/swagger/searchservice.json]: This one too?

File: search/2015-02-28-Preview/swagger/searchservice.json (hunk 1 above)
Comment (brjohnstmsft) [inline search/2015-02-28-Preview/swagger/searchservice.json]: Done.

**Comment by tbombach** [comment]
Merged

**Pull request merged**